amethyst_renderer/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
-name = "amethyst_renderer"
-version = "0.5.0"
-authors = ["Eyal Kalderon <ebkalderon@gmail.com>"]
+name = "amethyst_renderer_new"
+version = "0.6.0"
+authors = ["Eyal Kalderon <ebkalderon@gmail.com>", "<scareaangel@gmail.com>"]
 description = "High-level rendering engine with multiple backends"
 exclude = ["examples/*"]
-keywords = ["game", "engine", "renderer", "3d", "amethyst"]
+keywords = ["game", "engine", "renderer", "3d", "amethyst", "gfx", "gfx_hal"]
 categories = ["rendering", "rendering::engine"]
 
-documentation = "https://www.amethyst.rs/doc/master/amethyst_renderer/"
+documentation = "https://docs.rs/amethyst_renderer/0.1.0/amethyst_renderer/"
```

Comment:
Is it a good thing to change this? Especially locking the version isn't a good thing IMO.

Reply:
Old link leads to 404

Reply:
I think `https://docs.rs/amethyst_renderer` will be ideal.

Reply:
I think the main crate has the correct link